Source/Core/UICommon/GameFile.cpp

```diff
@@ -350,6 +355,30 @@ bool GameFile::IsElfOrDol() const
   return extension == ".elf" || extension == ".dol";
 }
 
+// Function for finding the folder name two (three if a Wii game) directories up for use in the file
+// name column for extracted games Returns a blank string if the folder can't be found
+std::string GameFile::FindFolderName() const
+{
+#ifdef HAS_STD_FILESYSTEM
```

Comment:
`HAS_STD_FILESYSTEM` was eliminated in #10994. You can rely on `std::filesystem` being available. The alternate implementation can be eliminated.